benchmarks/distributed/rpc/parameter_server/servers/AverageParameterServerBase.py

```diff
@@ -0,0 +1,14 @@
+from abc import abstractmethod
+
+from .ParameterServerBase import ParameterServerBase
+
+
+class AverageParameterServerBase(ParameterServerBase):
```

Comment:
Curious, is there a parameter server that is not averaging gradients

Reply:
From my limited reading, it seems that all parameter servers average gradients. I'll remove the AverageParameterServerBase class.